helprequest/migrations/0003_helprequest_slug.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.4 on 2021-07-07 03:48
+
+import autoslug.fields
```

Comment:
Is this a new requirement? If yes, can you add it on the requirements.txt, and update the README as of why we need this and how we're using it?

Reply:
hey...so I need to delete it... was trying to use it to make the URL work [update view] but it kind of failed...it's no longer a requirement